### DISCUSSION TITLE
SecurityException about high loader versions

### DISCUSSION THREAD

**Discussion by Sekakou** [FabricMC/fabric-api#2447] [Mod Dev Support] [answered]
I always meet SecurityException when lanuching Minecraft through the gradle task when the loader version is higher than 1.13.2(that's to say 1.14.x).What's more,when I back the loader version to 1.13.2 it works properly.
Someone guessed it may have something to do with the download of the jar was incomplete,but I tried delete the cache and gradle --refresh-dependencies,but them didn't work.
How can I solve it?Or where can I get the complete jar file?
![image](https://user-images.githubusercontent.com/100136138/183246171-4577789e-118f-44ce-b638-625301d80c94.png)

**Comment by modmuss50** [comment, marked as answer]
What loom version are you using? You can find it at the top of your build.gradle file?
